fe/lib/components/ConnectionPagination.tsx

```diff
@@ -0,0 +1,88 @@
+import {
```

Comment:
If we're not planning to export `ConnectionPagination` and `CopyableOid`, they should live under the `/private` subdirectory.